packages/dep-check/src/profiles/profile-0.61.ts

```diff
@@ -75,6 +75,14 @@ const profile: Profile = {
     name: "@react-native-masked-view/masked-view",
     version: "^0.2.4",
   },
+  "babel-preset-react-native": {
+    name: "metro-react-native-babel-preset",
+    version: "^0.56.0",
+  },
+  metro: {
+    name: "metro",
+    version: "^0.56.0",
+  },
```

Comment:
Can you add `devOnly: true` so these don't get propagated to the consuming apps?
```suggestion
  "babel-preset-react-native": {
    name: "metro-react-native-babel-preset",
    version: "^0.56.0",
    devOnly: true,
  },
  metro: {
    name: "metro",
    version: "^0.56.0",
    devOnly: true,
  },
```

Reply:
sure, will do!